cmake/match.py

```diff
@@ -76,7 +77,14 @@ def main():
 
     with open(input_file, 'r') as input, open(match_file, 'r') as match:
         input_lines = input.readlines()
-        match_lines = match.readlines()
+        # Filter out comments (lines beginning with the comment character,
+        # ignoring leading whitespace)
+        match_lines = list(
```

Comment:
I wonder, is it worth filtering out empty lines as well, just so we can group things more logically?

```
# Foo doesn't work
foo_test
foo2_test

# Bar doesn't work
bar_test
```

Reply:
That's a good idea, yeah. Thanks!